starter-code/src/app/contact-list/contact-list.component.ts

```diff
@@ -17,8 +17,7 @@ export class ContactListComponent implements OnInit {
   }
 
   addContact(){
-    console.log("Add contact has been called");
-    // add contact to contacts list
-    // clear inputs
+    contactList.push(this.newContact)
+    this.newContact = "";
```

Comment:
If the contact is of type Object, you shouldn't set it back to an empty string, but maybe to `{}`